src/main/java/com/github/timm/cucumber/generate/CucumberITGenerator.java

```diff
@@ -188,9 +186,14 @@ private String createFormatStrings() {
 
         for (int i = 0; i < formatStrs.length; i++) {
             final String formatStr = formatStrs[i].trim();
-            sb.append(String.format("\"%s:%s/%s.%s\"", formatStr,
-                            config.getCucumberOutputDir().replace('\\', '/'), fileCounter,
-                            formatStr));
+
+            if ("pretty".equalsIgnoreCase(formatStr)) {
+                sb.append("\"pretty\"");
```

Comment:
Poor assumption on my part. I was under the impression that the 'pretty' plugin did not require a :path nor would it do anything with it. It looks like it operates like any other plugin; if a path is specified, cucumber will direct the plugin output to that location, if not, it will direct it to the console. If you use multiple plugins, only one can be directed to the console; all others have to have a :path.

Looks like the best way to handle cucumber plugins is to add an option to the parallel plugin to a single cucumber plugin to be directed to the console if desired. What do you think?